src/index.html

```diff
@@ -9,6 +9,8 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Colored blocks</h1>
+    <div id = "red"></div>
```

Comment:
```suggestion
    <div class="red"></div>
```
remove redundant spaces